mrp_production_auto_post_inventory/tests/test_mrp_production.py

```diff
@@ -4,51 +4,50 @@
 import odoo.tests.common as common
 
 
-class TestMrpProductionAutoPost(common.SavepointCase):
+class TestMrpProductionAutoPost(common.TransactionCase):
```

Comment:
I need to change this to TransactionCase otherwise it won't pass test because of method _cron_post_inventory() which make a database commit.